.circleci/config.yml

```diff
@@ -10,12 +10,27 @@ parameters:
   releases-slack-channel:
     type: string
     default: dps-releases
+  java-version:
+    type: string
+    default: "17.0"
+
+executors:
+  hmpps/java:
+    parameters:
+      jdk_tag:
+        type: string
+        default: << pipeline.parameters.java-version >>
+  hmpps/node:
+    parameters:
+      node_tag:
+        type: string
+        default: "16.15-browsers"
```

Comment:
Prior to https://github.com/ministryofjustice/hmpps-circleci-orb/pull/116 this was set to `14.16-browsers` but I thought I'd bump to latest